src/manage.c

```diff
@@ -268,13 +268,14 @@ static u16 manage_warm_start(gnss_signal_t sid, const gps_time_t* t,
     } else {
       const almanac_t *a = &almanac[sid_to_global_index(sid)];
       if (a->valid) {
-        calc_sat_az_el_almanac(a, t->tow, t->wn-1024,
```

Comment:
It appeared to be a crude way to handle GPS rollover. Currently GPS is in the second cycle of GPS weeks, thus the "true" WN is some number greater than 1024, but the over the wire WN in GPS nav data is mod 1024.

The ephemeris functions correctly handle this, it appeared the old almanac functions did not.